### ISSUE TITLE
Venue ID's are invalid & date search endpoint breaks valid ID's.

### ISSUE THREAD

**Issue by catley85** [seanmtracey/CineList-API#21]
Example
- Endpoint `https://api.cinelist.co.uk/get/times/cinema/7530` functions correctly and returns listings for that cinema in JSON.
  - Adding the date query as `https://api.cinelist.co.uk/get/times/cinema/7530?day=21022018` returns error below.
```
{
"status": "ERR",
"reason": "Not a valid cinema ID"
}
```
- Other various cinema ID's also return the same error. Example `https://api.cinelist.co.uk/get/times/cinema/9105` which around 6 months ago was working now returns the error.

**Comment by seanmtracey** [comment]
The day offset should be the number of days from now. Today is `?day=0` tomorrow is `?day=1` and so on up to a week (6).

**Comment by seanmtracey** [comment]
Looking into it, I think a patch I've released (#15) has caused your error. The CineList API was never meant to take a timestamp as a value for getting different days, the fact that it worked was a quirk that I worked to undo. My bad, I should documented that change better.

The error message you received could definitely be better, or I suppose I could allow timestamps to be part of the API `YYYY-MM-DD`, but I really don't want to get into string parsing.

Happy to accept a PR if you're willing to put one together?
